.haml-lint.yml

```diff
@@ -1,4 +1,7 @@
 linters:
+  InlineStyles:
+    enabled: false
```

Comment:
I want to be able to specify background images in haml.

Reply:
lol ❤️